modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -39,23 +45,40 @@ sudo apt install openssl
 
 # PHP extensions needed to use ownCloud
 sudo apt install php-mysql php-mbstring php-gettext php-intl php-redis \
-     php-imagick php-igbinary php-gmp php-curl php-gd php-zip php-imap \
-     php-ldap php-bz2 php-phpseclib
+     php-imagick php-igbinary php-gmp php-bcmath php-curl php-gd php-zip \
+     php-imap php-ldap php-bz2 php-phpseclib
 ----
 
+== Useful Commands Managing PHP Extensions 
+
+=== List Enabled PHP Extensions
+ 
 If you want to retrieve a list of enabled PHP extensions run following command:
 
 [source,console]
 ----
 ls `php -i | grep "^extension_dir" | sed -e 's/.*=> //'`
 ----
 
+=== Enabling and Disabling PHP Extensions
+
+To enable or disable a PHP extension, use the commands `phpenmod` or `phpdismod`.
+
+Example:
+[source,console]
+----
+sudo phpenmod php-ldap
+ or
+sudo phpdismod php-ldap
+----
+
 == HASH Message-Digest Framework
 
 This framework does not need manual intervention anymore because:
 
-* as of PHP 5.1.2, the Hash extension is bundled and compiled into PHP by default.
-* as of PHP 7.4.0, the Hash extension is a core PHP extension, so it is always enabled.
+* As of PHP 5.1.2, the Hash extension is bundled and compiled into PHP by default.
+* It may be explicitly disabled by using the --disable-hash switch to configure
+* As of PHP 7.4.0, the Hash extension is a core PHP extension, so it is always enabled.
```

Comment:
What to do about mentioning PHP 7.4 ?
ownCloud does not yet support PHP 7,4, so we do not want people to accidentally think it will work.

Reply:
removed the complete section